.travis.yml

```diff
@@ -18,6 +15,14 @@ matrix:
     include:
         - php: 5.3
           env: COMPOSER_FLAGS="--prefer-lowest"
+          dist: precise
+        - php: 7.0
+          before_script:
+            - curl -sSfL -o ~/.phpenv/versions/7.0/bin/phpunit https://phar.phpunit.de/phpunit-5.7.phar
```

Comment:
It didn't occur to me when I suggested you create a PR that the PHPUnit dependency would be a problem. I think this can be rewritten to our approach in [mongo-php-library](https://github.com/mongodb/mongo-php-library/blob/master/.travis.yml). That entails adding alternatives for PHPUnit to `require-dev` and changing the tests themselves to use the namespaced class names.